lib/builders/Input.Chat.js

```diff
@@ -68,12 +81,12 @@ function addReply (replyDiv) {
   const qChatMessage = replyDiv
     .addChildTag('q-chat-message')
     .addAttribute(':name', 'reply.createdBy')
+    .addAttribute(':stamp', `formatDate(reply.created, 'DD/MM/YY HH:mm')`)
     .addAttribute(':key', 'replyIdx')
     .addAttribute('sent', null)
 
   qChatMessage
     .addChildTag('div')
-    .addAttribute('class', 'q-px-sm')
```

Comment:
This line was adding padding to reply comments which was inconsistent with regular comments, and looked strange once the stamp was added (as seen below):
![reply_before](https://github.com/wmfs/cardscript-to-quasar/assets/75633865/616509d4-fe0e-4513-b194-675a59fb5213)
Without this line, the text aligns with the stamp and is consistent with regular comments:
![reply_after](https://github.com/wmfs/cardscript-to-quasar/assets/75633865/3bed0c7b-1d57-4fb6-9394-f90406400d59)